lib/infrastructure/templates/renderTemplates/access-request-email/email/html.ejs

```diff
@@ -122,13 +122,17 @@
 
                 <p>To access services for <%=orgName%> please contact the approver at the organisation.</p>
             <%} else {%>
-                <p>Your DfE Sign-in account request to link <%=orgName%> has been rejected.</p>
-
-                <p>You can submit a new request. Check you have the correct organisation and provide information, such as, your role at the organisation, or the services you will need.</p>
+                <p> Your request to connect to <%=orgName%> has been rejected.</p>
+            <% if (reason) { %>
+               <p> The rejection reason is: <%=reason%></p>
+            <% } %>
+            <h4> What should I do next? </h4>
+                <p> Submit an amended request, or request access to another organisation. </p>
+            <p> If submitting an amended request, we recommend including information that will help identify you, such as your role. </p>
             <% } %>
 
             <p>Kind regards</p>
-            <p>DfE Sign-in</p>
+            <p><b>DfE Sign-in</b></p>
```

Comment:
Can you leave this not bold to be consistent with the other emails we send out?

Reply:
sure